ecosystem/sep-0031.md

```diff
@@ -594,32 +594,33 @@ On success the response should include a `200 OK` HTTP status code and the follo
+| `started_at`              | UTC ISO 8601 string | (optional) Start date and time of transaction.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `updated_at`              | UTC ISO 8601 string | (optional) The date and time of transaction reaching the current `status`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `completed_at`            | UTC ISO 8601 string | (optional) Completion date and time of transaction.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `user_action_required_by` | UTC ISO 8601 string | (optional) The date and time by when the user action is required. In certain statuses, such as `pending_customer_info_update`, anchor waits for the user action and `user_action_required_by` field should be used to show the time anchors gives for the user to make an action before transaction will automatically be moved into a different status (such as `expired` or to be `refunded`). `user_action_required_by` should only be specified for statuses where user action is required, and omitted for all other. Anchor should specify the action waited on using message or `more_info_url`. |
```

Comment:
SEP-31 involves two users, how should this field be used in this context? For the sender, for both?

Reply:
I think we can do it for both users, as it's makes things more simple